tools/generate_prefs.xsl

```diff
@@ -400,10 +401,10 @@
   </xsl:template>
 
   <xsl:template match="dtconfig[type='dir']" mode="tab">
-		<xsl:text>    widget = gtk_file_chooser_button_new(_("select directory"), GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER);
-		gtk_file_chooser_button_set_width_chars(GTK_FILE_CHOOSER_BUTTON(widget), 20);
-		gtk_widget_set_hexpand(widget, TRUE);
-		gtk_widget_set_halign(widget, GTK_ALIGN_FILL);
+    <xsl:text>    widget = gtk_file_chooser_button_new(_("select directory"), GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER);
```

Comment:
tabs indents converted to spaces - sorry, Geany is pig-headed about that

Reply:
In fact that's a fix :) We don't want tabs in code, thanks.